Build failure: handbrake

### Steps To Reproduce
Build `handbrake` (1.8.2) with `ffmpeg-full` (7.0.2)

### Build log
<details>

```
error: builder for '/nix/store/8fsbv1wpjv51izb4w01n0dw1n4hnqx2y-ffmpeg-full-7.0.2.drv' failed with exit code 2;
       last 25 log lines:
       > CC   libavcodec/libtwolame.o
       > CC     libavcodec/libvo-amrwbenc.o
       > CC libavcodec/libvorbisdec.o
       > CC   libavcodec/libvorbisenc.o
       > CC   libavcodec/libvpxdec.o
       > CC      libavcodec/libvpxenc.o
       > CC      libavcodec/libwebpenc.o
       > CC     libavcodec/libwebpenc_animencoder.o
       > CC libavcodec/libwebpenc_common.o
       > CC      libavcodec/libx264.o
       > CC        libavcodec/libx265.o
       > CC        libavcodec/libxavs.o
       > CC        libavcodec/libxevd.o
       > CC        libavcodec/libxeve.o
       > CC        libavcodec/libxvid.o
       > libavcodec/libxeve.c: In function 'get_conf':
       > libavcodec/libxeve.c:199:27: error: incompatible types when assigning to type 'XEVE_RATIONAL' {aka 'struct _XEVE_RATIONAL'} from type 'long int'
       >   199 |         cdsc->param.fps = lrintf(av_q2d(avctx->framerate));
       >       |                           ^~~~~~
       > libavcodec/libxeve.c: In function 'libxeve_encode':
       > libavcodec/libxeve.c:484:40: error: incompatible types when assigning to type 'int' from type 'XEVE_RATIONAL' {aka 'struct _XEVE_RATIONAL'}
       >   484 |                 avpkt->time_base.den = xectx->cdsc.param.fps;
       >       |                                        ^~~~~
       > make: *** [ffbuild/common.mak:81: libavcodec/libxeve.o] Error 1
       > make: *** Waiting for unfinished jobs....
       For full logs, run 'nix log /nix/store/8fsbv1wpjv51izb4w01n0dw1n4hnqx2y-ffmpeg-full-7.0.2.drv'.
error: 1 dependencies of derivation '/nix/store/z24d33jbrw5nrznzj5mql2y4byvz3rdz-handbrake-1.8.2.drv' failed to build

```
</details>

### Additional context

<details>
<summary>flake.lock</summary>

```
...
    "nixpkgs": {
      "locked": {
        "lastModified": 1730449684,
        "narHash": "sha256-Hlv3rTPxnO+DpKRXw9yjzERLdk05h7+fEbZxWM2taCw=",
        "owner": "nixos",
        "repo": "nixpkgs",
        "rev": "ab464abbeb3a2833288c6e907488c49c2e599f88",
        "type": "github"
      },
      "original": {
        "owner": "nixos",
        "ref": "nixos-unstable-small",
        "repo": "nixpkgs",
        "type": "github"
      }
    },
...
```

</details>

### Notify maintainers

<!--
Please @ people who are in the `meta.maintainers` list of the offending package or module.
If in doubt, check `git blame` for whoever last touched something.
-->

@Anton-Latukha
@wmertens

### Metadata

```console
[user@system:~]$ nix-shell -p nix-info --run "nix-info -m"
 - system: `"x86_64-linux"`
 - host os: `Linux 6.11.5, NixOS, 24.11 (Vicuna), 24.11.20241029.6a1c602`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Lix, like Nix) 2.91.1
System type: x86_64-linux
Additional system types: i686-linux, x86_64-v1-linux, x86_64-v2-linux, x86_64-v3-linux
Features: gc, signed-caches
System configuration file: /etc/nix/nix.conf
User configuration files: /home/ulad/.config/nix/nix.conf:/nix/store/2r8y4w48a22gi95c70i9awy0naanhr39-cinnamon-settings-daemon-6.2.0/etc/xdg/nix/nix.conf:/nix/store/2r8y4w48a22gi95c70i9awy0naanhr39-cinnamon-settings-daemon-6.2.0/etc/xdg/nix/nix.conf:/etc/xdg/nix/nix.conf:/home/ulad/.nix-profile/etc/xdg/nix/nix.conf:/nix/profile/etc/xdg/nix/nix.conf:/home/ulad/.local/state/nix/profile/etc/xdg/nix/nix.conf:/etc/profiles/per-user/ulad/etc/xdg/nix/nix.conf:/nix/var/nix/profiles/default/etc/xdg/nix/nix.conf:/run/current-system/sw/etc/xdg/nix/nix.conf
Store directory: /nix/store
State directory: /nix/var/nix
Data directory: /nix/store/n99cjrvzmw28v6vy6nbkcp0niwvlhwnb-lix-2.91.1/share`
```

---

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/development/libraries/ffmpeg/generic.nix
+++ b/pkgs/development/libraries/ffmpeg/generic.nix
@@ -135,8 +135,8 @@
 , withXcbShape ? withFullDeps # X11 grabbing shape rendering
 , withXcbShm ? withFullDeps # X11 grabbing shm communication
 , withXcbxfixes ? withFullDeps # X11 grabbing mouse rendering
-, withXevd ? withFullDeps && lib.versionAtLeast version "7" && !xevd.meta.broken # MPEG-5 EVC decoding
-, withXeve ? withFullDeps && lib.versionAtLeast version "7" && !xeve.meta.broken # MPEG-5 EVC encoding
+, withXevd ? withFullDeps && lib.versionAtLeast version "7.1" && !xevd.meta.broken # MPEG-5 EVC decoding
+, withXeve ? withFullDeps && lib.versionAtLeast version "7.1" && !xeve.meta.broken # MPEG-5 EVC encoding
 , withXlib ? withFullDeps # Xlib support
 , withXml2 ? withFullDeps # libxml2 support, for IMF and DASH demuxers
 , withXvid ? withHeadlessDeps && withGPL # Xvid encoder, native encoder exists

